pkg/storage/disk/getter.go

```diff
@@ -0,0 +1,35 @@
+package disk
+
+import (
+	"io/ioutil"
+	"path/filepath"
+	"time"
+
+	"github.com/gomods/athens/pkg/storage"
+)
+
+func (v *storageImpl) Get(baseURL, module, vsn string) (*storage.Version, error) {
```

Comment:
nit: vsn is almost unclear, and version isn't too many more characters.  Feel free to ignore